go.mod

```diff
@@ -2,4 +2,8 @@ module github.com/GoogleCloudPlatform/grpc-gcp-tools
 
 go 1.13
 
-require google.golang.org/grpc v1.25.1
+require (
+	github.com/vishvananda/netlink v1.0.0
```

Comment:
usage of this dependency looks good and should help when adding portability to non-linux OS's like windows!